segmentstore/contracts/src/main/java/io/pravega/segmentstore/contracts/tables/TableStore.java

```diff
@@ -61,6 +65,18 @@ default int maximumValueLength() {
         return 1040384; // 1MB - maximumKeyLength();
     }
 
+    /**
+     * Generates a set of {@link AttributeUpdate}s that set the initial Attributes on a newly create Table Segment.
+     *
+     * Attributes:
+     * * {@link Attributes#TABLE_INDEX_OFFSET} is initialized to 0.
+     *
+     * @return A Collection of {@link AttributeUpdate}s.
+     */
+    static Collection<AttributeUpdate> getInitialTableAttributes() {
```

Comment:
I will be fixing this mess with #3273 so that I don't have to pollute this interface with these details.